tests/unit/test_simulator.py

```diff
@@ -0,0 +1,93 @@
+"""
+Tests for simulated data
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import ga4gh.datamodel.variants as variants
+
+
+class TestSimulatedVariantSet(unittest.TestCase):
+    """
+    Test properties of the SimulatedVariantSet
+    """
+    def setUp(self):
+        self.randomSeed = 0
+        self.numCalls = 1
+        # ensure variantDensity is >= 1 so we get deterministic behavoir
+        self.variantDensity = 2
+        self.variantSetId = 3
+        self.simulatedVariantSet = self._getSimulatedVariantSet()
+        self.referenceName = 'ref'
+        self.startPosition = 100
+        self.endPosition = 103
+        self.variantName = 'unused'
+        self.callSetIds = ['unused']
+        self.bases = ["A", "C", "G", "T"]
+
+    def _getSimulatedVariantSet(self):
+        simulatedVariantSet = variants.SimulatedVariantSet(
+            self.randomSeed, self.numCalls,
+            self.variantDensity, self.variantSetId)
+        return simulatedVariantSet
+
+    def _getSimulatedVariantsList(self, simulatedVariantSet=None):
```

Comment:
Nope, still not seeing where this parameter is used. It doesn't have any effect anyway, even if it is used: you don't use `simulatedVariantSet` anywhere in the function.